### PR TITLE
feat(web): Syslumenn auctions via Syslumenn API - 2

### PR DESCRIPTION
**Pull request by siggifv**
# Syslumenn auctions via Syslumenn API - 2

Note that this feature was previously hot-fixed into `release/11.0.0` (see #4864) but had to be reverted (see #4873) as it turned out the feature was not fully ready. 

## What

Now that the Syslumenn API is production ready, we can deploy this new Syslumenn Auctions page, which fetches auctions data via Syslumenn API instead of fetching it from Contentful, saving Syslumenn staff manual labor maintaining the data in Contentful, which can be highly stressful moments before auctions take place.
We’ve also made some improvements in the UI/UX in the Auctions page since our last try to deploy this feature.
This PR cherry-picks the following to the release branch:
- PR #4822 
- PR #4823
- PR #4921 
- Commit b81a6995f1287465f6cb738bf60c984c888e41c9 - Minor fix for time formatting in UI.

## Why

As stated above, this relieves stress on Syslumenn staff, as they don't need to manually maintain the auction data in Contentful. Instead the auction data is fetched directly from their systems via the Syslumenn API.

## Screenshots / Gifs

syslumenn-auctions

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/Organization/Syslumenn/Auctions.tsx [hunk 1]
```diff
@@ -2,46 +2,390 @@
...
+  // Norðurland vestra
+  {
+    filterLabel: 'Norðurland vestra',
+    slugValue: 'syslumadurinn-a-nordurlandi-vestra',
+    office: 'Sýslumaðurinn á Norðurlandi vestra',
+    location: '',
+  },
+  {
+    filterLabel: ' - Blönduós',
+    slugValue: 'syslumadurinn-a-nordurlandi-vestra-bolnduos',
```
Comment (kksteini) [inline apps/web/screens/Organization/Syslumenn/Auctions.tsx:143]: "bolnduos" - Á þetta ekki að vera blonduos?

**Review by kksteini — DISMISSED** [review]
LGTM

File: apps/web/screens/Organization/Syslumenn/Auctions.tsx (hunk 1 above)
Comment (siggifv) [reply apps/web/screens/Organization/Syslumenn/Auctions.tsx]: Jú, takk 👍  Búinn að lagfæra þetta.

**Review by ingolfurs — APPROVED** [review]

**Review by kksteini — APPROVED** [review]
LGTM

**Pull request merged**